src/network/local_socket_reply.cpp

```diff
@@ -252,17 +254,33 @@ void mp::LocalSocketReply::send_request(const QNetworkRequest& request, QIODevic
 
 void mp::LocalSocketReply::read_reply()
 {
-    auto data_ptr = reply_data.data();
+    auto data_ptr = reply_data.data() + reply_offset;
     int bytes_read{0};
 
     do
     {
-        bytes_read = local_socket->read(reply_data.data(), len);
+        if (reply_offset + local_socket->bytesAvailable() > len)
```

Comment:
Instead of an error, we could resize the `QByteArray` instead?  `len` is really just an arbitrary size.

Reply:
Done.